src/commands/generateRunbook.ts

```diff
@@ -114,12 +125,18 @@ export const generateRunbook = (secrets: vscode.SecretStorage) => vscode.window.
         let Username = DEFAULT_USER;
 
         if (auth.stdout.toString().startsWith("{") && auth.status === 0 && !auth.error) {
-            const authPayload = JSON.parse(auth.stdout.toString()) as {
-                "ServerURL": string,
-                "Username": string,
-                "Secret": string
-            };
-            Username = authPayload.Username;
+            try {
+                const authPayload = JSON.parse(auth.stdout.toString()) as {
+                    "ServerURL": string,
+                    "Username": string,
+                    "Secret": string
+                };
+                Username = authPayload.Username;
+            }
+            catch (e) {
+                throw new Error(`Expected JSON from docker-credential-desktop, got STDOUT: ${auth.stdout.toString()} STDERR: ${auth.stderr.toString()} ERR: ${(auth.error || "N/A").toString()}`);
```

Comment:
There are no known cases where we successfully running docker-credential-desktop and get non-json - this seems like it'd have to be a corrupt install so this message is for "us", not for users, right?
I'm sure this is fine but is this message being shown to users anywhere?

Reply:
It can return `Credentials cannot be found in native keychain`

Reply:
got it